### ISSUE TITLE
Within each ecosystem list all standaradized ways to build a project with Nix

### ISSUE THREAD

**Issue by malteneuss** [NixOS/nix-book#28]
For example, to build a Haskell project there are several established ways, which are scattered along the docs, blog posts and so forth. It would be good to have an overview page of all these ways, like e.g. https://discourse.nixos.org/t/super-simple-haskell-development-with-nix/14287/3.

**Comment by fricklerhandwerk** [comment]
Yes, there should absolutely be an overview on all supported languages. The `nixpkgs` manual has a fair amount of information, but only for supported builders.
